parsers/s01-parse/timokoessler/gitlab-logs.yaml

```diff
@@ -0,0 +1,30 @@
+onsuccess: next_stage
+filter: "evt.Parsed.program == 'gitlab'"
+name: timokoessler/gitlab-logs
+description: "Parse GitLab Logs"
+pattern_syntax:
+  GITLAB_FAILED_PASSWORD: ^{"method":"POST","path":("\/users\/sign_in"|"\/users\/auth\/ldapmain\/callback").*("status":0|"action":"failure").*"time":"%{TIMESTAMP_ISO8601:timestamp}".*{"login":"%{USERNAME:username}".*"remote_ip":"%{IP:source_ip}".*
```

Comment:
Parsing JSON with regexp seems a little fragile.
We have a `JsonExtract` helper that will parse JSON, it would be better to use it.
The doc is [here](https://docs.crowdsec.net/docs/next/expr/helpers/#jsonextractjsonblob-fieldname-string) and a usage example [here](https://github.com/crowdsecurity/hub/blob/1945256760861906cf1515e0192f60c444dd83cf/parsers/s01-parse/crowdsecurity/fastly-logs.yaml) (you can also just search for `JsonExtract` in this repository to see all the usage)

Reply:
Okay, I will do that.
How can I combine multiple JsonExtract as a condition to set log_type?